### ISSUE TITLE
experimental syntax 'exportDefaultFrom' isn't currently enabled 

### ISSUE THREAD

**Issue by FrancisVila** [signavio/react-mentions#305]
**Steps to reproduce:**
npx create-react-app react-mentions
npm install --save react-mentions
download example files and copy to /src
npm start
error message: 
```
SyntaxError: C:\react\react-mentions3\react-mentions3\src\higher-order\index.js: Support for the experimental syntax 'exportDefaultFrom' isn't currently enabled (1:8):
> 1 | export provideExampleValue from './provideExampleValue'
    |        ^
  2 |
Add @babel/plugin-proposal-export-default-from (https://git.io/vb4yH) to the 'plugins' section of your 
```
went to https://git.io/vb4yH, did as instructed
`npm install --save-dev @babel/plugin-proposal-export-default-from`
error persists

**Comment by frontendphil** [comment]
Thanks for reporting. We'll need to look into this but I concur that this lib should probably not rely on that experimental behaviour.

**Comment by jfschwarz** [comment]
fixed with #308
